DhawaAlkhreej/DhawaAlkhreej/Modules/Calculator/CalculatorVC.swift

```diff
@@ -10,34 +10,81 @@ import UIKit
 
 class CalculatorVC: BaseViewController {
 
+    // MARK: Labels
     @IBOutlet var ThanwiyahLabel: UILabel!
     @IBOutlet var quodratLabel: UILabel!
     @IBOutlet var tahsilyLabel: UILabel!
     @IBOutlet var stepExamLabel: UILabel!
+    @IBOutlet var calculatedPercentageLabel: UILabel!
+    
+    // MARK: Buttons
     @IBOutlet var calculateButton: UIButton!
-    @IBOutlet var resultLabel: UILabel!
     
+    // MARK: Requsted Textfields
     @IBOutlet var requestedThanawiyahTF: DesignableTF!
+    @IBOutlet var requestedQuodratTF: DesignableTF!
+    @IBOutlet var requestedTahsilyTF: DesignableTF!
+    @IBOutlet var requestedStepExamTF: DesignableTF!
+    
+    // MARK: Accomplished Textfields
+    @IBOutlet var accoplishedThanawiyahTF: DesignableTF!
+    @IBOutlet var accomplishedQuodratTF: DesignableTF!
+    @IBOutlet var accomplishedTahsilyTF: DesignableTF!
+    @IBOutlet var accomplishedStemExamTF: DesignableTF!
     
-    var percantage = ["30%", "40%", "50%"]
-    var selectedPercentage: String?
+    // MARK: Class Variables
+    var percantages = ["٠٪", "١٠٪", "٢٠٪", "٣٠٪", "٤٠٪", "٥٠٪", "٦٠٪"]
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        setupUI()
```

Comment:
No need to call `setupUI()` in `viewDidLoad()` here because it’s already been called in the `BaseViewController`